### ISSUE TITLE
Confusion about Example 2

### ISSUE THREAD

**Issue by theres-waldo** [WICG/scroll-boundary-behavior#21]
There's something that doesn't make sense to me about Example 2:

> A page wants to implement their own pull-to-refresh effect and thus needs to disable browser native overscroll action.
> 
> ```
> html {
>   /* only disable pull-to-refresh but allow swipe navigations */
>   scroll-boundary-behavior-y: contain;
> }
> 
> ```
> In this case, the author can use contain on the viewport defining element to prevent overscroll from triggering navigation actions.

The stated objective is to "disable browser native overscroll action", but the suggestion is to use `contain`, which only disabled non-local boundary actions, and overscroll effects are defined to be local boundary actions. Shouldn't `none` be used in this case?

The comment "only disable pull-to-refresh but allow swipe navigations" doesn't make sense to me either. Pull-to-refresh is not a browser behaviour to be disabled to begin with, and swipe navigations, being a non-local boundary action, would be disabled by either `contain` or `none`.

Clarification of this example would be appreciated.

**Comment by theres-waldo** [comment]
> swipe navigations, being a non-local boundary action, would be disabled by either `contain` or `none`.

I think I get this now: since swipe navigation is a _horizontal_ boundary action, setting only `scroll-boundary-behavior-y: contain` (while leaving `scroll-boundary-behavior-x` at its default value of `auto`) leaves swipe navigation intact.

**Comment by theres-waldo** [comment]
I also understand now that "pull to refresh" _is_ a boundary default action that some browsers (such as Chrome for Android) do.

So I think I was just misreading "browser native overscroll action" - I thought it meant the visual overscroll effect (which is "local"), but it looks like it's intended to actually mean the native pull-to-refresh behaviour, if present (which is "non-local").

So, in light of that, the example does make sense. The only suggestion I would perhaps make is to rephrase "browser native overscroll action" with "browser native pull-to-refresh, if present".

**Comment by bgirard** [comment]
Yes, your understanding is correct. Thank you for the suggested clarification. I'll prepare a commit for it.

**Comment by majido** [comment]
@bgirard Did your commit land? should we close?
